platform/bitcoin/model.go

```diff
@@ -27,7 +27,7 @@ type Transaction struct {
 	Vin           []Output `json:"vin"`
 	Vout          []Output `json:"vout"`
 	BlockHash     string   `json:"blockHash"`
-	BlockHeight   uint64   `json:"blockHeight"`
+	BlockHeight   int64    `json:"blockHeight"`
```

Comment:
why int64? negative block number?

Reply:
Yes, when transaction is pending block hight = `-1`